site/source/user-manual/service-guides/vaultwarden/bitwarden-walkthrough.rst

```diff
@@ -0,0 +1,457 @@
+.. _bitwarden-walkthrough:
+
+=========================
+Vaultwarden Initial Setup
+=========================
+
+.. contents::
+  :depth: 2 
+  :local:
+
+.. _vaultwarden-webvault:
+
+.. collapse:: Web Vault
+
+    Web Vault
+    ---------
+    #. First we will open up our Start9 Server and go to the Marketplace tab. Select **Vaultwarden**, and install if you have not already done so:
+
+        .. figure:: /_static/images/services/vaultwarden/vaultwarden-in-marketplace.png
+                    :width: 50%
+                    :alt: vaultwarden-in-marketplace
+
+        .. figure:: /_static/images/services/vaultwarden/vaultwarden-install.png
+                    :width: 50%
+                    :alt: vaultwarden-install
+
+    #. Once installed click **Configure**:
+    
+        .. figure:: /_static/images/services/vaultwarden/vaultwarden-needs-config.png
+                    :width: 40%
+                    :alt: vaultwarden-needs-config
+
+    #. A strong admin token has been securely created by your Server, but you may change it if you wish. This can also be changed later. Click **Save**:
+
+        .. figure:: /_static/images/services/vaultwarden/vaultwarden-save-config.png
+                    :width: 50%
+                    :alt: vaultwarden-save-config
+    
+    #. Click start to run the service:
+
+        .. figure:: /_static/images/services/vaultwarden/vaultwarden-start-service.png
+                    :width: 50%
+                    :alt: vaultwarden-start-service
+
+    #. Once the health check turns green, click **Launch UI**:
+    
+        .. figure:: /_static/images/services/vaultwarden/vaultwarden-launch-ui.png
+                    :width: 50%
+                    :alt: vaultwarden-launch-ui
+
+    #. Time to create an account! Click **Create account**:
+    
+        .. note:: You are creating an account with yourself on your own Vaultwarden website served from your own Vaultwarden server - there are no third parties involved here.  
+            
+        .. figure:: /_static/images/services/vaultwarden/vaultwarden-create-account.png
+                    :width: 50%
+                    :alt: vaultwarden-create-account
+
+    #. Enter an email address for login:
+
+        .. figure:: /_static/images/services/vaultwarden/vaultwarden-create-account-email.png
+                    :width: 50%
+                    :alt: vaultwarden-create-account-email
+    
+        .. tip:: This email address can be anything you like. It doesn't have to be real. It is simply a way for you to log into Vaultwarden from Bitwarden apps on your devices. Your Vaultwarden server won't ever email you.
```

Comment:
This is true, however there may be advantages to using real emails - the OS will soon be able to email you, and this already works to some extent, importantly with the latest Vaultwarden update

Reply:
Could you elaborate? Is it able to send email now? Can it be used to reset passwords?

Reply:
These are questions for the Vaultwarden channel - sorry I do not know the details and have not used these features myself

Reply:
I'm inclined to leave this, I considered posting a link to official docs but they're pretty bad.